pos_loyalty/models/pos_order.py

```diff
@@ -24,7 +24,7 @@ def create_from_ui(self, orders):
         res = super(PosOrder, self).create_from_ui(orders)
         for order in orders:
             order_partner = order['data']['partner_id']
-            order_points = order['data']['loyalty_points']
+            order_points = order['data'].get('loyalty_points', 0)
```

Comment:
@legalsylvain Are you sure that this part should be in this branch ?

Reply:
This is a trivial patch of pos_loyalty (Travis fail with the current PR, without this patch, when I create an order during the test of pos_picking_delayed) but I can revert it and make it in another PR to make the cherry pick working more easy, you're right.

Reply:
Done in https://github.com/OCA/pos/pull/294.